tests/lib/network_helper.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jan 23 10:45:21 2019
+
+@author: gerben
+"""
+
+from netaddr import *
+import socket
+import struct
+import re
+class mac_custom(mac_unix):
+    pass
+
+def mac_format(mac_addr):
+    mac_tmp = re.sub("[\s+\.\!\-\:\/_,$%^*(+\"\')]+|[+()?【】“”！，。？、~@#￥%……&*（）]+',","",mac_addr)
+    mac_custom.word_fmt = '%.2X'
```

Comment:
I'm sorry, I try code forget to delete it.
And I will add upper() inside it

Reply:
No no... if you NEED it it's fine! I was just making sure you were doing this because you needed it. This example was verbatim from the documentation so I was not sure.

Reply:
I look lib to check, it was repeat, so I modify next patch and test.
It work same as before.
Then, I try it use upper and lower, it also could do, but I think almost mac type are upper, it can be eazy to use at future.